compiler/ast.nim

```diff
@@ -223,6 +223,7 @@ type
     nkBreakState,         # special break statement for easier code generation
     nkFuncDef,            # a func
     nkTupleConstr         # a tuple constructor
+    nkConstTuple,         # a ``const (a, b) = expr`` construct
```

Comment:
``macros.nim`` needs to be patched too.

Reply:
maybe we should add a test that makes sure these stay in sync
(doesnt' have to be in this PR though)